### DISCUSSION TITLE
O que estão achando das discussions do github?

### DISCUSSION THREAD

**Discussion by marcosdosea** [marcosdosea/gcm-ufs#400] [General]
Eu achei uma boa ferraenta para cadastro de ideias que ainda não sabemos se vai dar tempo implementar.

**Comment by marcosdosea** [comment]
E para registro de ideias para melhoria do processso de desenvovlimento.

**Reply by NathanaelSantos** [reply]
Também concordo!

**Reply by idylicaro** [reply]
NICE!

**Reply by NathanaelSantos** [reply]
Tenha a encomenda lá, vai querer?

**Comment by joannestephany** [comment]
Deveras interessante só não entendi ainda.
